oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/SelectParentItem.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from "react";
+import { RemoteSelectField } from "react-invenio-forms";
+import { getIn, useFormikContext } from "formik";
+// import { RemoteSelectField } from "./RemoteSelectField";
+import { http } from "react-invenio-forms";
+// for testing purposes
+
+const serializeSuggestions = (suggestions) =>
+  suggestions.map((item) => ({
+    text: item.title?.cs,
+    value: item.id,
+    key: item.id,
+  }));
+
+export const SelectParentItem = ({ vocabularyRecord }) => {
+  const { values } = useFormikContext();
+  const [initialParent, setInitialParent] = useState("");
+  const [loading, setLoading] = useState(false);
+  console.log(initialParent);
+  let getParentInfoApiUrl;
+  if (vocabularyRecord.hierarchy?.parent) {
+    getParentInfoApiUrl = `/api/vocabularies/institutions/${vocabularyRecord.hierarchy.parent}`;
```

Comment:
vocabulary type shouldn't be hardcoded here, use form config context & `record.links`